build.sh

```diff
@@ -103,39 +186,48 @@ else
-  rm .wattsi-output.zip
+  unzip $($VERBOSE && echo "-v" || echo "-qq") $HTML_TEMP/wattsi-output.zip -d $HTML_TEMP/wattsi-output
+  rm $HTML_TEMP/wattsi-output.zip
 fi
 
-rm -f index
-cat .wattsi-output/index-html | perl .post-process-index-generator.pl | perl .post-process-partial-backlink-generator.pl > index;
+rm -f $HTML_OUTPUT/index
+cat $HTML_TEMP/wattsi-output/index-html | perl .post-process-index-generator.pl | perl .post-process-partial-backlink-generator.pl > $HTML_OUTPUT/index;
 
 # multipage setup
-ln -s ../images .wattsi-output/multipage-html/
-ln -s ../link-fixup.js .wattsi-output/multipage-html/
-ln -s ../entities.json .wattsi-output/multipage-html/
+ln -s ../images $HTML_TEMP/wattsi-output/multipage-html/
+ln -s ../link-fixup.js $HTML_TEMP/wattsi-output/multipage-html/
+ln -s ../entities.json $HTML_TEMP/wattsi-output/multipage-html/
 
-rm -rf multipage
-mv .wattsi-output/multipage-html multipage
+rm -rf $HTML_OUTPUT/multipage
+mv $HTML_TEMP/wattsi-output/multipage-html $HTML_OUTPUT/multipage
+
+cp -p  $HTML_SOURCE/.htaccess $HTML_OUTPUT
+cp -p  $HTML_SOURCE/404.html $HTML_OUTPUT
+cp -pR $HTML_SOURCE/fonts $HTML_OUTPUT
+cp -pR $HTML_SOURCE/images $HTML_OUTPUT
+cp -pR $HTML_SOURCE/link-fixup.js $HTML_OUTPUT
 
 $QUIET || echo
 $QUIET || echo "Checking for potential problems..."
 # show potential problems
 # note - would be nice if the ones with \s+ patterns actually cross lines, but, they don't...
-grep -ni 'xxx' source | perl -lpe 'print "\nPossible incomplete sections:" if $. == 1'
-egrep -ni '( (code|span|var)(>| data-x=)|[^<;]/(code|span|var)>)' source | perl -lpe 'print "\nPossible copypasta:" if $. == 1'
-grep -ni 'chosing\|approprate\|occured\|elemenst\|\bteh\b\|\blabelled\b\|\blabelling\b\|\bhte\b\|taht\|linx\b\|speciication\|attribue\|kestern\|horiontal\|\battribute\s\+attribute\b\|\bthe\s\+the\b\|\bthe\s\+there\b\|\bfor\s\+for\b\|\bor\s\+or\b\|\bany\s\+any\b\|\bbe |be\b\|\bwith\s\+with\b\|\bis\s\+is\b' source | perl -lpe 'print "\nPossible typos:" if $. == 1'
-perl -ne 'print "$.: $_" if (/\ban (<[^>]*>)*(?!(L\b|http|https|href|hgroup|rb|rp|rt|rtc|li|xml|svg|svgmatrix|hour|hr|xhtml|xslt|xbl|nntp|mpeg|m[ions]|mtext|merror|h[1-6]|xmlns|xpath|s|x|sgml|huang|srgb|rsa|only|option|optgroup)\b|html)[b-df-hj-np-tv-z]/i or /\b(?<![<\/;])a (?!<!--grammar-check-override-->)(<[^>]*>)*(?!&gt|one)(?:(L\b|http|https|href|hgroup|rt|rp|li|xml|svg|svgmatrix|hour|hr|xhtml|xslt|xbl|nntp|mpeg|m[ions]|mtext|merror|h[1-6]|xmlns|xpath|s|x|sgml|huang|srgb|rsa|only|option|optgroup)\b|html|[aeio])/i)' source | perl -lpe 'print "\nPossible article problems:" if $. == 1'
-grep -ni 'and/or' source | perl -lpe 'print "\nOccurrences of making Ms2ger unhappy and/or annoyed:" if $. == 1'
-grep -ni 'throw\s\+an\?\s\+<span' source | perl -lpe 'print "\nException marked using <span> rather than <code>:" if $. == 1'
+grep -ni 'xxx' $HTML_SOURCE/source| perl -lpe 'print "\nPossible incomplete sections:" if $. == 1'
```

Comment:
Interesting. Just noticed these all operate on the source. We could (future PR) separate them out into a separate script, call it "lint.sh".